packages/remix/src/build.ts

```diff
@@ -196,6 +198,157 @@ export const build: BuildV2 = async ({
       const rootDirectory = relative(repoRootPath, workPath);
       serverBuildPath = join(rootDirectory, serverBuildPath);
     }
+
+    if (remixConfig.vercel?.images) {
+      images = remixConfig.vercel.images;
+    }
+
+    if (images) {
+      if (typeof images !== 'object') {
+        throw new Error(
+          `remix.config.js "images" should be an object received ${typeof images}.`
+        );
+      }
+
+      if (!Array.isArray(images.domains)) {
+        throw new Error(
+          `remix.config.js "images.domains" should be an Array received ${typeof images.domains}.`
```

Comment:
Should all of these error messages be `vercel.images.*` now?

Reply:
Nice catch! they are now.